src/Angor/Client/Pages/Create.razor

```diff
@@ -125,20 +150,69 @@
                     <label class="form-label">The amount in % to allocate for stage @(project.ProjectInfo.Stages.IndexOf(stage) + 1)</label>
                     <div class="input-group">
                         <InputNumber @bind-Value="stage.AmountToRelease" class="form-control" placeholder="Enter amount to release as a percentage" min="1" max="100" step="1" />
-                        <InputDate @bind-Value="stage.ReleaseDate" class="form-control"/>
+                        <InputDate @bind-Value="stage.ReleaseDate" class="form-control" />
                         <button type="button" class="btn btn-danger" @onclick="() => RemoveStage(stage)">Remove</button>
                     </div>
                 </div>
             }
+
             <button type="button" class="btn btn-primary mb-3" @onclick="AddStage">Add Stage</button>
 
             <!-- Save & Publish Buttons -->
             <div class="mb-3">
-                <button type="submit" class="btn btn-success">Next</button>
+                <button type="submit" class="btn btn-success" disabled="@(createProfileSpinner || project.NostrApplicationSpecificDataCreated)">
+                    @if (createApplicationDataSpinner)
+                    {
+                        <span class="spinner-border spinner-border-sm" role="status" aria-hidden="true"></span>
+                        <span>Next...</span>
+                    }
+                    else
+                    {
+                        <span>Next</span>
+                    }
+                </button>
             </div>
+
         </EditForm>
     }
 
+    @if (activeTab == 3)
```

Comment:
Should we show the metadata as well as part of the read only tab?

Reply:
yeah I will add the name and about for now